.gitlab/image.yml

```diff
@@ -1,82 +1,83 @@
 .use_gko-nocuda-gnu9-llvm8:
-  image: localhost:5000/gko-nocuda-gnu9-llvm8
+  image: ginkgohub/cpu:gnu9-llvm8
   tags:
     - private_ci
     - cpu
+    - amd
 
 .use_gko-nocuda-gnu9-llvm8-intel:
-  image: localhost:5000/gko-nocuda-gnu9-llvm8
+  image: ginkgohub/cpu:gnu9-llvm8-intel2020
   tags:
     - private_ci
     - cpu
-    - cuda
+    - controller
 
 .use_gko-cuda90-gnu5-llvm39:
-  image: localhost:5000/gko-cuda90-gnu5-llvm39
+  image: ginkgohub/cuda:90-gnu5-llvm39
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
```

Comment:
Is this and the other cuda jobs supposed to be `cpu`

Reply:
Yes, because we cannot run on GPUs with the current settings (it's also tied to `default_build`).